column_transforms/rasgo_cast/rasgo_cast.sql

```diff
@@ -0,0 +1,7 @@
+SELECT
+*
+{%- for col_expr, new_type in new_cols %}
+    -- TODO: offer an option for inplace or new column for each change?
```

Comment:
new columns is fine for now - we've followed that pattern throughout. i'll add a drop columns UDT to the list so we can drop unwanted ones explicitly. eventually we probably make more of them work in place but for now, not a big deal.